tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-training-apis.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           --enable_training_apis \
           --use_cuda --cuda_version=11.6 --cuda_home=/usr/local/cuda-11.6 --cudnn_home=/usr/local/cuda-11.6 \
           --build_wheel \
+          --build_java \
```

Comment:
This change here will only build the training api for java. But will not run the unit tests considering we have `--skip_tests`. Is it possible to run training specific tests (using some keyword filters) from https://github.com/microsoft/onnxruntime/blob/main/orttraining/orttraining/test/python/orttraining_test_ort_apis.py?

We can get this in another PR if this is not straightforward.

Reply:
The Java tests are run by gradle, and need a specific set of command line arguments to enable the right EP tests and to point at the native library (as when the tests are run the jar package hasn't been fully created yet). Plus they need to be run in the right directory so they can reference the appropriate onnx test models which live elsewhere in the source tree outside the `java` folder.

So it's a bit of a pain to do, and I wasn't sure how to make the necessary modifications to that python as I'm not sure what environment it's running in.

Reply:
I think it might be worth having another meeting to discuss how to integrate the tests, it'll probably be easier than doing it over email.

Reply:
Sounds good. We can talk about this over a call. I think we should not block this PR on that though.